spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.feature 'Show' do # rubocop:disable Metrics/BlockLength
+  let!(:user) do
+    new_user = User.new(
+      id: 1,
+      name: 'John Doe',
+      email: 'john.doe+2@example.com',
+      bio: "Hello, I'm John!",
+      posts_counter: 0,
+      photo: Rack::Test::UploadedFile.new(
+        Rails.root.join('app', 'assets', 'images', 'profile.jpg'),
+        'image/jpeg'
+      )
+    )
+    new_user.save
+    new_user
+  end
+
```

Comment:
Good job having the set-up for the post show page, however, you still have a couple of tests to write such as

- Tests that assert you can see the post's title.
- Tests that asserts you can see how many likes it has.
- Tests that asserts you can see the post body.
- Tests that asserts you can see the username of each commentor.
- Tests that asserts you can see the comment each commentor left.

consider adding the above mentioned test in order to get the specs complete for this page